app/src/main/java/a/a/a/kv.java

```diff
@@ -0,0 +1,117 @@
+package a.a.a;
+
+import android.animation.Animator;
+import android.content.Context;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.besome.sketch.beans.ProjectLibraryBean;
+import com.besome.sketch.editor.manage.library.firebase.FirebaseActivity;
+import com.google.android.material.materialswitch.MaterialSwitch;
+import com.sketchware.remod.R;
+
+import mod.SketchwareUtil;
+import mod.hey.studios.util.Helper;
+
+public class kv extends LinearLayout implements nv {
```

Comment:
Hey, it seems that you have decompiled this class. It'd be better if you first commit the decompiled file (without any changes) and refactor/fix it in another separate commit. That makes it easier to debug issues that might have been caused by decompiling later on. 
Thanks